### PR TITLE
Add Action Cable guide to list

### PR DESCRIPTION
**Pull request by maclover7**
This effectively publishes the "Action Cable Overview" guide. If we
don't think this is ready for the prime time, we can mark it as a "work
in progress" guide.

[ci skip]

### CODE REVIEW

File: guides/source/documents.yaml [hunk 1]
```diff
@@ -139,6 +139,10 @@
       name: Using Rails for API-only Applications
       url: api_app.html
       description: This guide explains how to effectively use Rails to develop a JSON API application.
+    -
+      name: Action Cable Overview
+      url: action_cable_overview.html
+      description: This guide explains Action Cable works, and how to use WebSockets to create real-time features.
```
Comment (prathamesh-sonpatki) [inline guides/source/documents.yaml]: explains `how`?

Comment (maclover7) [inline guides/source/documents.yaml]: 👍

**Comment by fxn** [comment]
Let me do an edit pass and we'll publish.

**Comment by maclover7** [comment]
👍

**Pull request merged**